### ISSUE TITLE
rpc_pb_service.js:249 Uncaught (in promise) Error: Response closed without headers

### ISSUE THREAD

**Issue by PostNZT** [filecoin-shipyard/powergate-pinning-service#2]
![image](https://user-images.githubusercontent.com/34854542/87234783-c1dfca80-c3fe-11ea-9eaf-7ed0dbdac1ce.png)
 
I already set the HOST to http://192.168.80.128:6001
![image](https://user-images.githubusercontent.com/34854542/87234813-08352980-c3ff-11ea-8f29-3e1677e7f1e9.png)

But I'm having an issue since on the network it only shows 0.0.0.0:6002

**Comment by vasa-develop** [comment]
The Powergate docker-compose scripts expose the powergate APIs on `0.0.0.0:6002`. So, you need to do this: `POW_HOST="http://0.0.0.0:6002"`.

In case you want to set it to any other IP:PORT (as you are trying to do), you will need to make changes to the `docker-compose-devnet.yaml` file.

See the line `6002:6002`. You will need to change it to `6001:6001`.

![powergate-file](https://user-images.githubusercontent.com/28847087/87515350-58c7b380-c699-11ea-9e1c-b246995399ba.png)

**Comment by PostNZT** [comment]
I was able to resolved it by creating this structure since the powergate docker is listening to 0.0.0.0:6002, so I decided to create an express app for it to interact with the powergate and my react-application.

![structure](https://user-images.githubusercontent.com/34854542/87588034-340d2380-c70d-11ea-81d6-1ab903399522.PNG)
 
So I was able to get a token from powergate. And the thing is, I am running the powergate in my VPS. 
![happy](https://user-images.githubusercontent.com/34854542/87588264-8a7a6200-c70d-11ea-9453-c0d523f45a02.PNG)

Thanks for the reply though. I will close this issue.
